components/slack/sources/new-interactions/new-interactions.mjs

```diff
@@ -0,0 +1,83 @@
+import common from "../common/base.mjs";
+
+export default {
+  name: "New Interactions",
+  version: "0.0.1",
+  key: "new-interactions",
+  description: "Emit new events on each new interactions",
```

Comment:
s/interactions/interaction

Reply:
Should also link to docs maybe? Since users will necessarily have to pair this source with a corresponding block kit action, right? (and similarly we should figure out how to reference this source in any Slack action that supports block messages?)

Reply:
Adding docs, but block message crafting is a _wholllllleeeeee_ other can of worms.